datachannel.go

```diff
@@ -101,7 +101,7 @@ func (c *DataChannel) Protocol() string {
 	return C.GoString(p)
 }
 
-func (c *DataChannel) MaxPacketLifeTime() uint {
+func (c *DataChannel) MaxRetransmitTime() uint {
```

Comment:
The internal libwebrtc property is still `maxRetransmitTime` but the spec defined API is `maxPacketLifeTime`, so maybe we should leave this alone?

https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/maxPacketLifeTime